node/clients/graphql-server.ts

```diff
@@ -0,0 +1,25 @@
+import { AppClient, GraphQLClient, InstanceOptions, IOContext, RequestConfig } from '@vtex/api'
+
+
+export class GraphQLServer extends AppClient {
+  protected graphql: GraphQLClient
+
+  constructor(ctx: IOContext, opts?: InstanceOptions) {
+    super('vtex.graphql-server@1.x', ctx, opts)
+    this.graphql = new GraphQLClient(this.http)
+  }
+
+  public query = async (query: string, variables: any, extensions: any, config: RequestConfig) => {
```

Comment:
extensions don't need to be `any`, right ?

Reply:
why can't we also make use of `generics` ?

Reply:
I didn't wanna restrict the extensions because I don't know if there are other extensions besides `persistedQuery`.